### PR TITLE
Add a CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by ingridcrant**
Adding a CONTRIBUTING.md to this repository as part of an effort to improve our open source documentation. Looking for reviews!

Here's what it looks like:
Screenshot 2024-01-25 at 9 58 59 AM

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
+# Contributing
+
+Hello and thank you for your interest in contributing to the 1Password `connect-helm-chart` project 👋! We sincerely appreciate the time and effort you put into improving our Helm Charts. Before you start, please take a moment to read through this guide to understand our contribution process.
```
Comment (jillianwilson) [inline CONTRIBUTING.md:3]: Personal opinion but I'd remove the "hello" here

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+## Getting Started
+
+To start contributing, you'll need to fork this repository and clone it locally. Next,
```
Comment (jillianwilson) [inline CONTRIBUTING.md:7]: Ton contirubute, folks won't actually need to fork the repo, they can just work directly from it :)

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+- Running `helm lint` in the applicable subdirectory will verify that your chart follows best practices.
+
+- To run `helm template --debug`:
+
+1. navigate to the root of the repository
+2. `cd..`
+3. `helm template connect-helm-charts/charts/secrets-injector --debug` for **secrets-injector** OR `helm template connect-helm-charts/charts/connect --debug` for **connect**.
+
+For more debugging templates, feel free to consult the [docs](https://helm.sh/docs/chart_template_guide/debugging/).
+
```
Comment (jillianwilson) [inline CONTRIBUTING.md:38]: I recommend adding a section about updating documentation if they add a new configuration, maybe we can point them towards the tables in the readme

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
+# Contributing
+
+Hello and thank you for your interest in contributing to the 1Password `connect-helm-chart` project 👋! We sincerely appreciate the time and effort you put into improving our Helm Charts. Before you start, please take a moment to read through this guide to understand our contribution process.
+
+## Getting Started
+
+To start contributing, you'll need to fork this repository and clone it locally. Next,
+
+1. Get the latest [Helm release](https://github.com/helm/helm#install).
+2. Add the 1password repository to be able to download and install all the charts from this repository:
```
Comment (jillianwilson) [inline CONTRIBUTING.md:10]: To contribute these steps 2-4 aren't necessary, as they will be working from their local copy

**Comment by ingridcrant** [comment]
New CONTRIBUTING.md visual:
Screenshot 2024-01-26 at 2 58 15 PM

**Comment by jillianwilson** [comment]
Looks good Ingrid!

**Review by jillianwilson — APPROVED** [review]

**Review by edif2008 — APPROVED** [review]
This contribution documentation looks great. Left some non-blocking comments. 😊

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+
+Thank you for your interest in contributing to the 1Password `connect-helm-chart` project 👋! We sincerely appreciate the time and effort you put into improving our Helm Charts. Before you start, please take a moment to read through this guide to understand our contribution process.
+
+## Getting Started
+
+To start contributing, get the latest [Helm release](https://github.com/helm/helm#install).
+
+## Testing
+
+NOTE: This only applies to changes made in `./charts/connect`.
```
[the remark this answers is not in the document]
Comment (ingridcrant) [reply CONTRIBUTING.md:11]: That's exactly it. We still test changes made to secrets-injector, but those are run when a pull request is made and waiting to merge.

**Pull request merged**